.git-blame-ignore-revs

```diff
@@ -1 +1 @@
-1be3f1bfb671fa6d59dcd2203f23e1625194574b
+8bf1c83817c2dce3576572715aac21e3fc9d6fa2
```

Comment:
Ugh? Why replace a previous one? Shouldn't this be a list? *confused*

Reply:
Initially I added a placeholder version, then when I amended/force-pushed the id of the commit changed :)